frontend/src/pages/Home.jsx

```diff
@@ -0,0 +1,18 @@
+
+import MenuBar from "../components/Header";
+import FormContainer from "../components/ContainerAligned";
+import HomePage from "../components/HomePage";
+function Home() {
+
+
+  return (
+    <nav>
+            <MenuBar/>
+            <HomePage/>
+
+    </nav>
```

Comment:
HomePage dans la navigation ?
```suggestion
    <nav>
        <MenuBar/>
    </nav>
    <main>
        <HomePage/>
    </main>
```

Reply:
kdo: https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantic_elements

Reply:
merciiii